products.py

```diff
@@ -1,62 +1,177 @@
+                    "vanilla candle", "citrus candle", "shovel", "planter pots", "soil", "strawberry seeds", "tomato seeds", "blackberry seeds"]
+
+music_list = ["big ben", "into the ocean", "clean slate", "demons", "Shrek", "The other woman", "Save Willy"]
+
+instrument_list =["guitar", "flute", "drums", "cello", "conga", "piano"]
+
+office_list = ['Paper','Pens','Pencils','Notebooks','Binders','Folders','Stapler','Staples','Paper Clips','Rubber Bands','Highlighters','Markers','Whiteboard','Whiteboard Markers','Eraser']
+
+outdoor_list = ['Camping Tent','Sleeping Bag','Camping Stove','Cooler','Backpack','Hiking Boots','Outdoor Clothing','Sleeping Pad','Lantern','First Aid Kit','Fishing Rod','Fishing Tackle',
+            'Binoculars','Campfire Grill','Multi-tool','Camping Chairs','Portable Water Filter','Map and Compass','Flashlight','Fire Starter','Sunscreen','Bug Spray','Water Bottle',
+            'Portable Charger']
+
+pet_list = ['Pet Food','Water Bowl','Food Bowl','Pet Bed','Leash','Collar','Harness','Grooming Brush','Cat Litter','Litter Box','Pet Carrier','Toys','Scratching Post','Pet Shampoo',
+            'Dental Chews','Pet Brush','Pet Nail Clippers','Pet Crate','Feeding Station','Pet Gates','Pet Cleaning Wipes','Training Pads','Pet Backpack','Pet Clothes','Pet Carrier Bag']
+
+software_list = ['Windows 11','macOS Ventura','Ubuntu','Fedora','Android','iOS', 'Adobe Photoshop']
+
+sports_list = ['Soccer Ball','Basketball','Baseball Glove','Baseball Bat','Volleyball','Tennis Racket','Golf Clubs','Golf Balls','Golf Tees','Golf Glove',
+            'Golf Bag','Rangefinder']
+
+toys_list = ['Soccer Ball','Basketball','Baseball Glove','Baseball Bat','Volleyball','Tennis Racket','Golf Clubs','Golf Balls','Golf Tees','Golf Glove',
+            'Golf Bag','Rangefinder']
+
+video_game_list = ['The Legend of Zelda: Breath of the Wild','Grand Theft Auto V','Red Dead Redemption 2','Assassin\'s Creed Valhalla','Spider-Man (PS4)','Horizon Forbidden West',
+                        'Call of Duty: Modern Warfare II','Battlefield 2042','DOOM Eternal','Overwatch 2','Counter-Strike: Global Offensive','Apex Legends']
+
+video_list = ['Mad Max: Fury Road','Inception','The Dark Knight','Avengers: Endgame','Jurassic Park','Mission: Impossible – Fallout','Star Wars: Episode IV - A New Hope',
+                        'Blade Runner 2049','The Matrix','Avatar','The Lord of the Rings: The Fellowship of the Ring','Guardians of the Galaxy']
+
+watch_list = ['Rolex Submariner','Omega Seamaster','Patek Philippe Nautilus','Audemars Piguet Royal Oak','Tag Heuer Carrera','Breitling Navitimer','Longines Master Collection',
+                    'Omega Speedmaster Professional','IWC Pilot’s Watch','Jaeger-LeCoultre Reverso','Cartier Tank','Tissot Le Locle']
+
+def generateProductId():
     num  = range(1,2000)
     productID = random.sample(num,1)[0]
```

Comment:
Random is a redundant import because faker already has an option to generate random numbers.
see [the docs](https://faker.readthedocs.io/en/stable/providers/baseprovider.html#faker.providers.BaseProvider.random_digit)

Reply:
We can use the pyint function and move forward